spectests/structs_encoding.go

```diff
@@ -1803,6 +1815,9 @@ func (a *AttesterSlashing) UnmarshalSSZ(buf []byte) error {
 
 	// Field (0) 'Attestation1'
 	{
+		if o0 < 8 {
```

Comment:
Why there is no offset check for 'Attestation2' in line 1832?

Reply:
My thinking was that after the first check, the second check incorporates the first offset as it's beginning slice, so all slice offsets are aligned to each other. If any offset after the first is lower than the previous offset, the go runtime will panic. We could explicitly check so that bad values can be rejected without causing a panic.

Reply:
Yeah on second thought I like explicit checks to avoid a runtime panic, I'm rewriting this code to check each offset now, thanks!